lib/HTTP/Status.pm

```diff
@@ -13,70 +13,82 @@ our @EXPORT_OK = qw(is_client_error is_server_error is_cacheable_by_default);
+# https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
 
 my %StatusCode = (
     100 => 'Continue',
     101 => 'Switching Protocols',
-    102 => 'Processing',                      # RFC 2518 (WebDAV)
-    103 => 'Early Hints',                     # RFC 8297
+    102 => 'Processing',                      # RFC 2518: WebDAV
+    103 => 'Early Hints',                     # RFC 8297: Indicating Hints
+#   104 .. 199
     200 => 'OK',
-    201 => 'Created',
+    201 => 'Created', 
     202 => 'Accepted',
     203 => 'Non-Authoritative Information',
     204 => 'No Content',
     205 => 'Reset Content',
-    206 => 'Partial Content',
-    207 => 'Multi-Status',                    # RFC 2518 (WebDAV)
-    208 => 'Already Reported',		      # RFC 5842
+    206 => 'Partial Content',                 # RFC 7233: Range Requests
+    207 => 'Multi-Status',                    # RFC 4918: WebDAV
+    208 => 'Already Reported',		          # RFC 5842: WebDAV bindings
+#   209 .. 225
+    226 => 'IM used',                         # RFC 3229: Delta encoding
+#   227 .. 299
     300 => 'Multiple Choices',
     301 => 'Moved Permanently',
     302 => 'Found',
     303 => 'See Other',
-    304 => 'Not Modified',
+    304 => 'Not Modified',                    # RFC 7232: Conditional Request
     305 => 'Use Proxy',
     307 => 'Temporary Redirect',
-    308 => 'Permanent Redirect',              # RFC 7238
+    308 => 'Permanent Redirect',              # RFC 7528: Permanent Redirect
+#   309 .. 399
     400 => 'Bad Request',
-    401 => 'Unauthorized',
+    401 => 'Unauthorized',                    # RFC 7235: Authentication
     402 => 'Payment Required',
     403 => 'Forbidden',
     404 => 'Not Found',
     405 => 'Method Not Allowed',
     406 => 'Not Acceptable',
-    407 => 'Proxy Authentication Required',
+    407 => 'Proxy Authentication Required',   # RFC 7235: Authentication
     408 => 'Request Timeout',
     409 => 'Conflict',
     410 => 'Gone',
     411 => 'Length Required',
-    412 => 'Precondition Failed',
+    412 => 'Precondition Failed',             # RFC 7232: Conditional Request
     413 => 'Request Entity Too Large',
     414 => 'Request-URI Too Large',
     415 => 'Unsupported Media Type',
-    416 => 'Request Range Not Satisfiable',
+    416 => 'Request Range Not Satisfiable',   # RFC 7233: Range Requests
     417 => 'Expectation Failed',
-    418 => 'I\'m a teapot',		      # RFC 2324
-    422 => 'Unprocessable Entity',            # RFC 2518 (WebDAV)
-    423 => 'Locked',                          # RFC 2518 (WebDAV)
-    424 => 'Failed Dependency',               # RFC 2518 (WebDAV)
-    425 => 'No code',                         # WebDAV Advanced Collections
-    426 => 'Upgrade Required',                # RFC 2817
-    428 => 'Precondition Required',
-    429 => 'Too Many Requests',
-    431 => 'Request Header Fields Too Large',
-    449 => 'Retry with',                      # unofficial Microsoft
+#   418 .. 420
```

Comment:
I don't think we should remove any pre-existing entries: someone might be using them.

Reply:
they were 'temporary' removed, to make the list IANA compliant ... intentionally
but I also planned to re add them.

I would not intentionally break people their stuff, quite aware that there is code out there in the wild.

But thanks for pointing out